native/src/modules/layout/containers/HeaderContainer.js

```diff
@@ -23,21 +24,35 @@ type StatePropsType = {|
   goToLanguageChange?: () => void,
   peeking: boolean,
   categoriesAvailable: boolean,
-  routeCityModel?: CityModel
+  routeCityModel?: CityModel,
+  shareUrl: string | null
 |}
 
 type DispatchPropsType = {| dispatch: Dispatch<StoreActionType> |}
 
 type PropsType = {| ...OwnPropsType, ...StatePropsType, ...DispatchPropsType |}
 
+const getShareUrl = (params: {| city: string, language: string, path: ?string, routeName: string |}): string | null => {
```

Comment:
Please add tests for this functionality if they are not yet present. Also again can't you say `?string` here?

Reply:
changed it to ?string and will add a test.